src/main/scala/ai/starlake/job/Main.scala

```diff
@@ -218,14 +219,29 @@ class Main extends StrictLogging {
           if (settings.appConfig.forceHalt) {
             Runtime.getRuntime.halt(1)
           }
```

Comment:
to include in preload job

Reply:
as discussed, needed if cmd called from an api